platform/ext/target/nordic_nrf/Native_Driver/spu.c

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
Can we move `Native_Driver` into `common`?

    nordic_nrf/common/Native_Driver

I think it makes more sense to keep everything that is common in the `common` directory.

Reply:
That for @anangl to decide, actually.

Reply:
Yes, let's place it in the common directory. I'd also suggest to name it `native_drivers` (we'll have there also `cmsis_drivers`).

Reply:
Sure. I am doing this on a separate commit to track the move.

Reply:
EDIT - done now (on a single commit)

Reply:
side note @anangl it's a bit messy across the different targets, regarding the nomenclature around cmsis and native driver directory names. Several targets use Native_Driver and CMSIS_Driver, while other use cmsis_drivers and native_drivers.